### ISSUE TITLE
Mi Band Demo NotFoundError

### ISSUE THREAD

**Issue by lkalneus** [beaufortfrancois/sandbox#5]
Hi! I've tried mi band demo, but got an error "NotFoundError: Service not found in device."

Should I turn something on  the device first before a try?

**Comment by beaufortfrancois** [comment]
Which OS do you run?
See https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md#chrome

**Comment by lkalneus** [comment]
Android 5.1.1 (OxygenOS 2.1.4, the device is Oneplus one). 
Also, I use Chromium 53.0.2754

**Comment by beaufortfrancois** [comment]
Let's figure out which one(s) is not found first ;)
Can you open the JS console (from `about:inspect`) and copy/paste these commands after trying to connect to it?

``` js
miband.server.getPrimaryService(0xFEE0).then(s => console.log('MIBAND_SERVICE'));
miband.server.getPrimaryService(0xFEE7).then(s => console.log('IOS_SERVICE'));
```

**Comment by lkalneus** [comment]
Well, the second one is failed. 

![image](https://cloud.githubusercontent.com/assets/2410911/15673897/82bced0e-275b-11e6-84a8-534b9a166365.png)

**Comment by beaufortfrancois** [comment]
Can you now install the [nRF Master Control Panel](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp&hl=en) Android app, scan for your MiBand device, connect to it and share found services?

**Comment by lkalneus** [comment]
![image](https://cloud.githubusercontent.com/assets/2410911/15674551/3ab94da0-275f-11e6-9eab-b67190c54e93.png)

**Comment by beaufortfrancois** [comment]
Do you have more services?
If not, do you know which firmware version of the MiBand device you're running?
Does your device work with the official [Mi Fit](https://play.google.com/store/apps/details?id=com.xiaomi.hm.health&hl=en) app?

**Comment by beaufortfrancois** [comment]
Oh! I wonder if it's because it is not the same model as well.
I've only tried this demo with my Mi Band device: http://www.amazon.com/gp/product/B00RMO0T5I

**Comment by lkalneus** [comment]
No, that's all. Yes it works with Mi Fit app. The firmware version is 4.16.3.7. Heart rate firmware version is 1.3.76.18

**Comment by lkalneus** [comment]
So, looks like there is no way for me to make this demo?

**Comment by beaufortfrancois** [comment]
You may want to try to remove https://github.com/beaufortfrancois/sandbox/blob/0f73ae87ba8ecd10c418ae733152a35c6d86a11e/web-bluetooth/mi-band/miBand.js#L65-L67 but even so, I think you need at some point to "pair" the device and you'll need the MAC Address for this IIRC ;(
